inc/api.class.php

```diff
@@ -271,6 +271,11 @@ protected function retrieveSession() {
             $_SESSION['glpi_use_mode'] = Session::NORMAL_MODE;
 
             Session::loadLanguage();
```

Comment:
Change code to before `Session::loadLanguage();`, to work plugins locales.

Reply:
I though the same but on the `inc/include.php` file the same `Session::loadLanguage();` is called before init the plugin, however your comment gave me the alert about there weren't tiggered the plugin's load & hook, so I'll fix that.